docs/how-to/craft/code/example-classic-confinement-autotools.yaml

```
@@ -0,0 +1,31 @@
+# autotools-classic-example, an example of a classically-confined project with Autotools
+
+name: autotools-classic-example
+version: "1.0"
+summary: An example snap for an autotools project
+description: |
+  Shows how to create a snap for an application that uses Autotools for
+  build and installation.
+
+grade: devel
+base: core22
+confinement: classic
+
+platforms:
```

My previous suggestion was wrong, sorry!  The make example will also need to be fixed.

```suggestion
architectures:
```

---

Fixed and rebased: [48535ce](https://github.com/canonical/snapcraft/pull/5345/commits/48535ce1030e3efde7552d3c48518876639f80f2)